packages/eslint-config-react-app/index.js

```diff
@@ -241,15 +224,15 @@ module.exports = {
     ],
 
     // https://github.com/benmosher/eslint-plugin-import/tree/master/docs/rules
-    'import/first': 'error',
-    'import/no-amd': 'error',
-    'import/no-webpack-loader-syntax': 'error',
+    'import/first': 'warn',
+    'import/no-amd': 'warn',
+    'import/no-webpack-loader-syntax': 'warn',
```

Comment:
These should stay errors. They're intentionally breaking the build for features we don't support.